pinot-core/src/test/java/org/apache/pinot/core/common/datatable/DataTableSerDeTest.java

```diff
@@ -426,6 +429,40 @@ public void testV2V3Compatibility()
     Assert.assertEquals(newDataTable.getMetadata(), EXPECTED_METADATA);
   }
 
+  @Test
+  public void testDataBlockBuilderV4()
```

Comment:
We should test `DataTableBuilder` here. `DataBlockBuilder` is used only for multi-stage engine, which is out of the scope of this test

Reply:
DataTableBuilder is already tested by `testV3V4Compatibility`